README.md

```diff
@@ -28,6 +28,21 @@ Include info the audience will find helpful:
 * Why might the audience use this?
 * Where can they find more?
 
+### What should I present with?
+
+This is up to you!  _Demo_ doesn't mean _live demo only_
+
+* Slides
+* Slides with PowerShell on them
+* The console
+* VS Code
+* PowerShell ISE
+* Pre-recorded video that you narrate
+* Notepad (maybe not)
+* Something else
+
+Just make sure you zoom enough for your audience to see
```

Comment:
nit:  Zoom in so it just fits in the whole screen.  Don't be shy.. Bigger is better!  You can't ask the audience if they can see

Reply:
@RamblingCookieMonster once this language is updated I'm happy to stamp and merge. :)

Reply:
Good call!  Added, thanks!